### PR TITLE
[CLOUD-3941] - Logger configurations in OCP

### PR DESCRIPTION
**Pull request by msmagnanijr**
See: https://issues.redhat.com/browse/CLOUD-3941

Signed-off-by: Mauricio Magnani <msmagnanijr@gmail.com>

### CODE REVIEW

**Comment by msmagnanijr** [comment]
@spolti could you please take a look? thanks!

**Comment by spolti** [comment]
lgtm, @luck3y please take a look when you have time.

File: jboss/container/wildfly/launch/json-logging/added/launch/json_logging.sh [hunk 1]
```diff
@@ -9,8 +9,10 @@ function configure_json_logging() {
   getConfigurationMode "##CONSOLE-FORMATTER##" "configureMode"
   if [ "${configureMode}" = "xml" ]; then
     configureByMarkers
+    configurePatternFormatterByMarkers
   elif [ "${configureMode}" = "cli" ]; then
     configureByCLI
+    configurePatternFormatterByCLI
```
Comment (jfdenise) [inline jboss/container/wildfly/launch/json-logging/added/launch/json_logging.sh:15]: This call will not have any effect. the generated CLI operations needs to be added to the CLI script file.
Should be: configurePatternFormatterByCLI >> ${CLI_SCRIPT_FILE}

**Comment by spolti** [comment]
@jfdenise  @luck3y can you guys please review it?

**Comment by luck3y** [comment]
@msmagnanijr Is there a version of this in master already?

**Comment by msmagnanijr** [comment]
> @msmagnanijr Is there a version of this in master already?

@luck3y  actually not yet.

**Comment by luck3y** [comment]
@msmagnanijr We'd generally need master first then cherry-picked to the branch. Otherwise this change will just get lost in the next version.

**Comment by msmagnanijr** [comment]
We'll take another approach. Thanks.

**Pull request closed without merging**